Framework/DataHandling/test/SetSampleTest.h

```diff
@@ -1135,4 +1135,4 @@ class SetSampleTest : public CxxTest::TestSuite {
   const std::string m_facilityName = "TEST_LIVE";
   const std::string m_instName = "ISIS_Histogram";
   const std::string m_envName = "TestEnv";
-};
+};
```

Comment:
Not sure what the GitHub diff is seeing here but can we remove this difference? Perhaps there's a carriage return missing in the new version